From: KevinDuan <duankaiwen@kylinos.cn>
Date: Wed, 19 Aug 2026 08:10:11 +0000
Subject: Translated using Weblate (Spanish)

Currently translated at 100.0% (40 of 40 strings)

Translation: openkylin-huanghe/ukui-panel
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-panel/es/
---
 panel/translations/panel_es.ts | 50 +++++++++++++++++++++---------------------
 1 file changed, 25 insertions(+), 25 deletions(-)

diff --git a/panel/translations/panel_es.ts b/panel/translations/panel_es.ts
index 1ac8d51..7430389 100644
--- a/panel/translations/panel_es.ts
+++ b/panel/translations/panel_es.ts
@@ -9,19 +9,19 @@
     </message>
     <message>
         <source>System Monitor</source>
-        <translation type="unfinished"></translation>
+        <translation>Monitor del sistema</translation>
     </message>
     <message>
         <source>Auto Hide</source>
-        <translation type="unfinished"></translation>
+        <translation>Ocultar automáticamente la barra de tareas</translation>
     </message>
     <message>
         <source>Panel Setting</source>
-        <translation type="unfinished"></translation>
+        <translation>Configuración de barra de tareas</translation>
     </message>
     <message>
         <source>Panel Size</source>
-        <translation type="unfinished"></translation>
+        <translation>Tamaño de barra de tareas</translation>
     </message>
     <message>
         <source>Large</source>
@@ -41,7 +41,7 @@
     </message>
     <message>
         <source>Panel Position</source>
-        <translation type="unfinished"></translation>
+        <translation>Posición de barra de tareas</translation>
     </message>
     <message>
         <source>Top</source>
@@ -61,22 +61,22 @@
     </message>
     <message>
         <source>Lock Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Bloquear barra de tareas</translation>
     </message>
     <message>
         <source>Show</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar</translation>
     </message>
     <message>
         <source>Switch to New Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Alternar barra de tareas de tres islas</translation>
     </message>
 </context>
 <context>
     <name>UkuiPanel::PanelNext</name>
     <message>
         <source>Panel Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Configuración de barra de tareas</translation>
     </message>
     <message>
         <source>Show Desktop</source>
@@ -84,39 +84,39 @@
     </message>
     <message>
         <source>System Monitor</source>
-        <translation type="unfinished"></translation>
+        <translation>Monitor del sistema</translation>
     </message>
     <message>
         <source>Data Island Form</source>
-        <translation type="unfinished"></translation>
+        <translation>Forma de isla de datos</translation>
     </message>
     <message>
         <source>Islands</source>
-        <translation type="unfinished"></translation>
+        <translation>Tres islas</translation>
     </message>
     <message>
         <source>Top Bar</source>
-        <translation type="unfinished"></translation>
+        <translation>Barra superior</translation>
     </message>
     <message>
         <source>Settings Island Form</source>
-        <translation type="unfinished"></translation>
+        <translation>Configurar forma de isla</translation>
     </message>
     <message>
         <source>Auto Hide</source>
-        <translation type="unfinished"></translation>
+        <translation>Ocultar automáticamente la barra de tareas</translation>
     </message>
     <message>
         <source>Lock Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Bloquear barra de tareas</translation>
     </message>
     <message>
         <source>Switch to Classical Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Alternar barra de tareas clásica</translation>
     </message>
     <message>
         <source>Panel Size</source>
-        <translation type="unfinished"></translation>
+        <translation>Tamaño de barra de tareas</translation>
     </message>
     <message>
         <source>Large</source>
@@ -139,7 +139,7 @@
     <name>UkuiPanel::PanelTopBar</name>
     <message>
         <source>Panel Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Configuración de barra de tareas</translation>
     </message>
     <message>
         <source>Show Desktop</source>
@@ -147,27 +147,27 @@
     </message>
     <message>
         <source>System Monitor</source>
-        <translation type="unfinished"></translation>
+        <translation>Monitor del sistema</translation>
     </message>
     <message>
         <source>Data Island Form</source>
-        <translation type="unfinished"></translation>
+        <translation>Forma de isla de datos</translation>
     </message>
     <message>
         <source>Islands</source>
-        <translation type="unfinished"></translation>
+        <translation>Tres islas</translation>
     </message>
     <message>
         <source>Top Bar</source>
-        <translation type="unfinished"></translation>
+        <translation>Barra superior</translation>
     </message>
     <message>
         <source>Settings Island Form</source>
-        <translation type="unfinished"></translation>
+        <translation>Configurar forma de isla</translation>
     </message>
     <message>
         <source>Switch to Classical Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Alternar barra de tareas clásica</translation>
     </message>
 </context>
 </TS>
